mcstatus/bedrock_status.py

```diff
@@ -8,6 +8,10 @@
 import asyncio_dgram
 
 from mcstatus.address import Address
+from mcstatus.status_response import BedrockStatusResponse
+
+# TODO Remove this useless __all__ after DEPRECATION_DATE
```

Comment:
Looks like the other conversation was marked by GitHub as outdated, so I resolved it to get it off the list. Even after reading that thread, I'm still wondering why this is useless.

Reply:
This is not useless **now**. But will be after the deprecation date. Like
```py
# before
__all__ = ["reexported_item"]
# and after
__all__ = []
```
The last one is the same as autogenerated `__all__`. There is no sense to keep `__all__` then.
---
> Well, I'm talking about pure python, no type checkers, what type checkers do about those imports is a different question, and indeed, mypy checks imports for presence in `__all__` (or using the default strategy if it's not defined), pyright doesn't. That's however not related to real functionality, and pyright is sort of right to not check for this presence, as that's how python would work on runtime.

In pure python, this does take [the small effect](https://stackoverflow.com/a/64130), that will be work even without `__all__` (autogenerated `__all__` also includes all imports). So `__all__` here is only for type checkers.